### PR TITLE
Adding support for urls and paths for addons

### PR DESCRIPTION
**Pull request by dahendel**
Need this for [rke issue](https://github.com/rancher/rke/issues/190).

### CODE REVIEW

File: apis/management.cattle.io/v3/rke_types.go [hunk 1]
```diff
@@ -11,6 +11,8 @@ type RancherKubernetesEngineConfig struct {
 	Authentication AuthnConfig `yaml:"authentication" json:"authentication,omitempty"`
 	// YAML manifest for user provided addons to be deployed on the cluster
 	Addons string `yaml:"addons" json:"addons,omitempty"`
+	// List of urls or paths for addons
+	AddonsInclude []string `yaml:"addons_include"`
```
Comment (HighwayofLife) [inline apis/management.cattle.io/v3/rke_types.go]: Missing json?

**Comment by dahendel** [comment]
Added json Tags

File: apis/management.cattle.io/v3/rke_types.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ type RancherKubernetesEngineConfig struct {
 	Authentication AuthnConfig `yaml:"authentication" json:"authentication,omitempty"`
 	// YAML manifest for user provided addons to be deployed on the cluster
 	Addons string `yaml:"addons" json:"addons,omitempty"`
+	// List of urls or paths for addons
+	AddonsInclude []string `yaml:"addons_include" json:"addons_include,omitempty"`
```
Comment (HighwayofLife) [inline apis/management.cattle.io/v3/rke_types.go]: Should be `addonsInclude,omitempty` format for json.

**Comment by galal-hussein** [comment]
@dahendel Thanks for the PR, the drone check is failing since you need to run `go generate` after adding this field

**Comment by dahendel** [comment]
Ahh ok, thanks I will run and push

**Comment by dahendel** [comment]
@galal-hussein  Done

**Comment by galal-hussein** [comment]
@dahendel Can you squash the commits into two commits one for the code change and one for the go generate

**Comment by alena1108** [comment]
@dahendel could you squash the commits into 2 - first with the real changes, second - the result of go generate

**Comment by dahendel** [comment]
@alena1108 @galal-hussein Done

**Pull request merged**